exe-common/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![type_length_limit = "2097152"]
```

Comment:
I wonder what monster type causes this.
What's the error if this directive is removed?

Reply:
holy 🐮 ! we have a type that needs 2 MB ?

Reply:
Hurray after a number of rewrites of the ntt-client, this requirement has gone.